src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -120,6 +121,8 @@ const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
 
       ...((bordered || v.borderColor || circular) &&
         getBorderedStyles(circular, v.borderColor || getIconColor(color, v))),
+
+      transform: `${rtl ? 'scaleX(-1)' : ''} rotate(${rtl ? -1 * rotate : rotate}deg)`,
```

Comment:
I love this this addition to Icon!! This is a personal preference, but I personally find it much easier to quickly understand what the transform is doing here if the code was written in multiple lines. eg)

```
...(rtl && {
    transform: `'scaleX(-1)' rotate(-1 * ${rotate}deg)`,
}),

...(!rtl && {
    transform: `rotate(${rotate}deg)`,
}),
```

Reply:
Agreed, will refactor it.